### ISSUE TITLE
Improve searching by adding suggestions

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#115]
## User story
1. As a User
2. I want to get usefull suggestions while typing search terms 
3. So that it is easier to complete the search

## Acceptance criteria
* Dynamic suggestions while searching (Address/coordinate, use external API)
* Search term completion via button press (tab, arrow keys)
* Option to still add information (e.g: street number) after selecting search term

## Additional
* Searching data points is done by filtering in the FE

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by op-hub** [comment]
Has been done already in previous issues: close this duplicate
